react/features/invite/components/dial-in-summary/native/DialInSummaryErrorDialog.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import React, { Component } from 'react';
+
+import { ErrorDialog } from '../../../../base/dialog';
+import { translate } from '../../../../base/i18n';
+import { connect } from '../../../../base/redux';
+
+/**
+ * Dialog to confirm a remote participant mute action.
+ */
+class DialInSummaryErrorDialog extends Component<{}> {
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        return (
+            <ErrorDialog
```

Comment:
I'm not sure I like this being called "ErrorDialog", as it has no relation with any error condition, except being called here. How about "AlertDialog"? Akin to the native UIAlertView (and whatever else on Android).